src/main/java/uk/gov/hmcts/reform/sscs/service/HmcHearingApi.java

```diff
@@ -35,4 +37,12 @@ HearingGetResponse getHearingRequest(
         @RequestParam("id") String id
     );
 
+    @DeleteMapping(value = "/hearing", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Is this change meant to be included in this PR?

Reply:
it just update from master, nothing I changed